esx_service/vmdk_ops_test.py

```diff
@@ -100,7 +100,8 @@ class VmdkCreateRemoveTestCase(unittest.TestCase):
     valid_attach_as_opt_1 = {volume_kv.ATTACH_AS: u'persistent'}
     valid_attach_as_opt_2 = {volume_kv.ATTACH_AS: u'independent_persistent'}
     name = ""
-    vm_name = 'test-vm'
+    random_id = random.randint(0, 65536)
```

Comment:
please separate this into a `generate_test_vm_name` function instead of cut-n-paste throughout the code. 
Also, replace these 2 lines in all places below the function call please